impls/msft-green-ai.yaml

```diff
@@ -0,0 +1,44 @@
+name: mft-green-ai
+description: "https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/msft-green-ai.md"
+tags:
+  kind: ml
+  complexity: simple
+  category: cloud
+config:
+  funit:
```

Comment:
Can see where you are heading here but we can handle this with a model that just divides by the funit and therefore this is just another model with config.

The sci model will divide the `carbon` value with anything else that looks like `r` and pump the result out as a value called `sci`